draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1626,12 +1626,22 @@ client uses `get-all-by-hash`, then it will already have the new STH.
 
 ### Evaluating compliance
 
-To be considered compliant, a certificate MUST be accompanied by at least one
-valid SCT. A certificate not accompanied by any valid SCTs MUST NOT be
-considered compliant by TLS clients.
+To be considered compliant with this specification, a certificate MUST be
+accompanied by evidence that it has been included in one or more logs. There
+are two types of proof that a client might require:
 
-A TLS client MUST NOT evaluate compliance if it did not send both the
-`transparency_info` and `status_request` TLS extensions in the ClientHello.
+1. A log inclusion proof to a Signed Tree Head
+2. A Signed Certificate Timestamp
+
+It is up to a client's local policy which forms of evidence are sufficient for
+compliance.  For example, a client might require certificates to come with some
+minimum number of SCTs from among a collection of trusted logs, or require a
+single inclusion proof to an audited tree head.
+
+Servers may provide evidence of logging via any of the channels described in
+{{tls_servers}}.  In order to ensure that any information sent by the server is
+successfully received by the client, clients that intend to evaluate compliance
+MUST support both the `transparency_info` and `status_request` TLS extensions.
```

Comment:
Yes, there's text in section 8.2.1. about clients having to support the three SCT delivery mechanisms - How about moving the last sentence into that section? 
Section 8.2.1. should explicitly say that TLS clients MUST support both the `transparency_info` and `status_request` TLS extensions for properly receiving SCTs, but it seems to me we should not force TLS clients to fully support processing the `status_request` / `status_request_v2` extensions (i.e. they could use it just for extracting SCTs).